src/Input/Silk.NET.Input.Common/Silk.NET.Input.Common.csproj

```diff
@@ -3,6 +3,7 @@
     <PropertyGroup>
         <TargetFramework>netstandard2.0</TargetFramework>
         <RootNamespace>Silk.NET.Input</RootNamespace>
+        <Nullable>enable</Nullable>
```

Comment:
Consider moving to top level file (and explicitly opting out where needed) so we can track this properly

Reply:
This probably isn't ideal as BuildTools has the capability to generate whole new projects.

Reply:
Fair enough, I'd expect BuildTools to generate `<Nullabe>disable</Nullable> but if it doesn't right now, it probably isn't worth it.